test/compilable/test16607.d

```diff
@@ -0,0 +1,15 @@
+struct A(T)
+{
+    T t; // causes A to be SIZEOKfwd b/c B (passed as T) isn't yet done
```

Comment:
> I'm fairly used to that style and have a lot of stuff to type each day.

IMHO it's a bad habit that ought to be avoided in general. I instantly know you're the author of any text when I see such an abbreviation, as I never see them anywhere else. If saving time is important, there exists software that can do that without impacting the text or readers. But maybe it's just me.